concepts/05 UI Components/Button/00 Getting Started with Button/08 Stylize the Button.md

```diff
@@ -0,0 +1,51 @@
+The Button supports several predefined color schemes. To apply one of them, set the [type](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#type) property. You can also use the [stylingMode](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#stylingMode) property to customize the fill and borders. Refer to the [Predefined Types](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) demo for more information about these properties.
```

Comment:
- "The Button supports several predefined color schemes" means what? You can apply different color schemes to the button?
- "To apply one of them" Why do you use 'one' instead of 'color scheme'? You should not use **one** or **ones** as a substitute